packages/url_launcher/url_launcher_android/pigeons/messages.dart

```diff
@@ -25,6 +27,12 @@ class WebViewOptions {
   final Map<String?, String?> headers;
 }
 
+class BrowserOptions {
```

Comment:
The class and property should have doc comments.

Reply:
Added those